src/script/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2023-present The Bitcoin Core developers
```

Comment:
Nit: it's kind of confusing that this is in `script/` and not in `consenus/`.

Reply:
Thanks! [Reworked](https://github.com/hebasto/bitcoin/pull/41#issuecomment-1857590536).